queries.sql

```diff
@@ -54,4 +54,14 @@ SELECT neutered, AVG(escape_attempts) AS average FROM animals GROUP BY neutered
 select species, min(weight_kg) as Minimun, max(weight_kg) as Maximum from animals
 GROUP BY species;
 SELECT species, AVG(escape_attempts) FROM animals 
-WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
\ No newline at end of file
+WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
+
+/* Pull request 3 */
+
+SELECT a.name FROM animals AS a JOIN owners AS b ON a.owner_id = b.id WHERE b.full_name = 'Melody Pond';
+ SELECT a.name FROM animals AS a JOIN species AS s ON a.species_id = s.id WHERE s.name = 'Pokemon';
+ SELECT a.name AS animal_name , o.full_name AS owner_name FROM animals AS a RIGHT JOIN owners As o ON a.owner_id = o.id;
+ SELECT COUNT(a.*), s.name FROM animals AS a JOIN species AS s ON a.species_id = s.id GROUP BY s.name;
+ SELECT a.name FROM animals AS a JOIN species AS s ON a.species_id = s.id JOIN owners as o ON a.owner_id = o.id WHERE o.full_name = 'Jennifer Orwell' AND s.name = 'Digimon';
+ SELECT a.name FROM animals AS a JOIN owners AS o ON a.owner_id = o.id WHERE a.escape_attempts = 0 AND o.full_name = 'Dean Winchester';
+ SELECT COUNT(a.owner_id) AS count, o.full_name FROM animals AS a JOIN owners AS o ON a.owner_id = o.id GROUP BY o.full_name  ORDER BY count DESC LIMIT 1;
```

Comment:
Good job on working on the project requirements. But it says in the project requirements that: Students should add a screenshots of their queries results to the pull request description because this helps us code reviewer sensure that students run the queries with success. So I t would be nice screen shots to all 7 queries to your PULL REQUEST.